pwiz_tools/Skyline/FileUI/PeptideSearch/TransitionSettingsControl.cs

```diff
@@ -349,11 +352,30 @@ public TransitionSettings GetTransitionSettings(Form parent)
                 settings.Instrument.MaxInclusions, settings.Instrument.MinTime, settings.Instrument.MaxTime);
             Helpers.AssignIfEquals(ref instrument, settings.Instrument);
 
-            TransitionLibraryPick pick = settings.Libraries.Pick != TransitionLibraryPick.none ? settings.Libraries.Pick : TransitionLibraryPick.all;
+            var pick = settings.Libraries.Pick != TransitionLibraryPick.none && DocumentSettings.PeptideSettings.Libraries.LibrarySpecs.Any(lib => !lib.IsDocumentLibrary)

```

Comment:
Add a comment explaining this conditional. I have to admit that "none" and "all" are not totally clear names.